lib/soapy_bing/ads/bulk/campaigns.rb

```diff
@@ -4,15 +4,19 @@ class Ads
     module Bulk
       class Campaigns
         DEFAULT_ENTITIES = %w(CampaignTargets Ads).freeze
-        POLLING_TRIES = 30
+        DEFAULT_POLLING_SETTINGS = {
+          tries: 20,
+          sleep: ->(n) { n < 7 ? 2**n : 120 }
+        }.freeze
```

Comment:
I'm not sure. This will be running in a worker, so I think it is not such big problem (I guess it is better to wait for 30-40 min than failing sooner). But we have to see how it goes in practice and make adjustments if needed.